### ISSUE TITLE
GenX.auto fails on TimeSpan

### ISSUE THREAD

**Issue by marklam** [cmeeren/fsharp-hedgehog-experimental#31]
```F#
module HedgehogExperimentalRepro.Tests

open System
open Hedgehog
open FsUnitTyped

[<Test>]
let ``can make timespan`` () =
    property {
        let! (test : TimeSpan) = GenX.auto
        test.Seconds |> shouldBeSmallerThan 100
    } |> Property.check
```

fails with the exception
```
System.OverflowException : Value was either too large or too small for an Int64.
at System.Numerics.BigInteger.op_Explicit(BigInteger value)
   at <StartupCode$Hedgehog>.$Numeric.FromBigInt@175-1.Invoke(BigInteger x)
   at Hedgehog.GenX.defaults@247-9.Invoke(Int32 sz)
   at Hedgehog.Gen.Int64@468-3.Invoke(Seed seed, Int32 size)
   at Hedgehog.Random.map@39-2.Invoke(Seed seed, Int32 size)
   at Hedgehog.Gen.bindRandom@40.Invoke(Seed seed0, Int32 size)
   at Hedgehog.Gen.bindRandom@40.Invoke(Seed seed0, Int32 size)
   at Hedgehog.Property.loop@332-21(Int32 n, Random`1 random, Seed seed, Int32 size, Int32 tests, Int32 discards)
   at Hedgehog.Property.Report(Int32 n, Property`1 p)
   at Hedgehog.Property.Check(Property`1 p)
   at HedgehogExperimentalRepro.Tests.can make timespan() 
```

**Comment by cmeeren** [comment]
Seems to be a Hedgehog bug: https://github.com/hedgehogqa/fsharp-hedgehog/issues/185

**Comment by moodmosaic** [comment]
Fixed in https://github.com/hedgehogqa/fsharp-hedgehog/pull/188, released as Hedgehog 0.8.3 :tada:
